addon/scheduler.ts

```diff
@@ -38,13 +38,20 @@ export function beginTransition(): void {
         });
       }).finally(() => {
         waiter.endAsync(scheduledWorkToken);
+        performance.mark('appSchedulerEnd');
+        performance.measure(
+          'appScheduler',
+          'appSchedulerStart',
+          'appSchedulerEnd'
+        );
       });
     });
   }
 }
 
 export function endTransition(): void {
   _whenRouteDidChange.resolve();
+  performance.mark('appSchedulerStart');
```

Comment:
Suggesting slightly different names:

```
performance.mark('app-scheduler.transition-completed')
performance.mark('app-scheduler.route-idle');
performance.measure(
  'app-scheduler.route-idle-delay',
  'app-scheduler.transition-completed',
  'app-scheduler.route-idle',
);
```

Reply:
I'm going to go ahead and merge this, as it matches what's in the 2.1.x branch. What you proposed here is structurally nicer, but I want to keep with the built-in standards, as I value that level of standardization.